finger/models.py

```diff
@@ -12,28 +13,87 @@ def update_data(self, data):
             if username and re.match('^[a-z]+$', username) and username != 'stacken':
+
+                phone_list = [entry.get('arbtelefon')] + \
+                             [entry.get('hemtelefon')]
+
+                comments_list = [entry.get('avdelning')] + \
+                                [entry.get('distrubution')] + \
+                                [entry.get('organisation')] + \
+                                [entry.get('status')] + \
+                                [entry.get('syssel')] + \
+                                [entry.get('comment')]
+                if entry.get('Utesluten'):
+                    comments_list = ["Utesluten"] + comments_list
+
+                if not entry.get('Fel_adress'):
+                    address_list = [entry.get('gatuadress')] + \
+                                   [entry.get('postadress')] + \
+                                   [entry.get('land')]
+                    if entry.get('c/o'):
+                        address_list = ["c/o " + entry.get('c/o')] + address_list
+                else:
+                    address_list = []
+
                 fields = {
+                    'title': entry.get('titel'),
                     'first_name': entry.get('förnamn'),
                     'last_name': entry.get('efternamn'),
                     'email': entry.get('mailadress'),
-                    'payed_year': entry.get('betalt', None),
-                    'ths_claimed_vt': entry.get('THS-studerande', None),
-                    'ths_claimed_ht': entry.get('THS-studerande', None),
+                    'payed_year': entry.get('betalt'),
+                    'ths_claimed_vt': entry.get('THS-studerande'),
+                    'ths_claimed_ht': entry.get('THS-studerande'),
+                    'phone': ", ".join(filter(None, phone_list)),
+                    'comments': "\n".join(filter(None, comments_list)),
+                    'address': "\n".join(filter(None, address_list)),
+                    'ths_name': entry.get('THS-namn'),
+                    'kth_account': entry.get('KTH-konto'),
+                    'has_key': entry.get('Hallnyckel', False),
+                    'support_member': entry.get('stödmedlem', False),
+                    'honourable_member': entry.get('Hedersmedlem', False),
+                    'keycard_number': entry.get('kortnr'),
                 }
                 if not fields.get('email'):
                     kthname = entry.get('KTH-konto')
                     if kthname:
                         fields['email'] = kthname + '@kth.se'
                     else:
                         fields['email'] = username + '@stacken.kth.se'
+
+                if entry.get('inträdesdatum'):
+                    fields['date_joined'] = parser.parse(entry.get('inträdesdatum'))
+                else:
+                    # Because I did not feel like to change the default type
+                    fields['date_joined'] = parser.parse("1970-01-01")
```

Comment:
Why not `parser.parse(entry.get('inträdesdatum', '1970-01-01'))` instead of the conditional?

Reply:
I moved things around, and did not realize that :) ... I will fix!